apps/vanilla/src/global/main.css

```
@@ -0,0 +1,337 @@
+/* Layout styles */
+
+.container {
+	display: flex;
+	justify-content: center;
+	padding: 0 var(--gap-large);
+}
+
+.main {
+	flex: 1;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	gap: var(--gap-medium);
+	max-width: var(--width-main-layout);
+}
+
+.content {
+	/* https://monosnap.com/file/vJ8gzEqXQhu77XzeCbj7djNiNQ2rj1 */
+	grid-template-columns: 0.66fr 0.06fr 0.28fr;
+	display: grid;
+	width: 100%;
+}
+
+/* Header styles */
+
+.main__header {
+	width: 100%;
+	height: var(--height-main-header);
+	padding: 20px 0;
+	border-bottom: 1px solid var(--shadow-color-regular);
+}
+
+/* Category styles */
+
+.category {
+	display: flex;
+	flex-direction: column;
+	gap: var(--gap-large);
+	overflow-y: scroll;
+
+	/* The reason we need to subtract gap 2 time -> [main-header] [gap] [content] [gap] [main-footer] */
+	height: calc(100vh - var(--height-main-header) - var(--height-main-footer) - 2 * var(--gap-medium));
+
+	/* This help hide scrollbar on Firefox, Safari and IE */
+	scrollbar-width: none;
+	-ms-overflow-style: none;
+}
+
+/* This help hide scrollbar on Chrome */
+.category::-webkit-scrollbar {
+	display: none;
+}
+
+.category__books {
+	display: flex;
+	flex-direction: column;
+}
+
+.category__books-header {
+	display: flex;
+	gap: var(--gap-small);
+	background-color: var(--main-background-color);
+	padding-bottom: var(--gap-medium);
+	position: sticky;
+	top: 0;
+}
+
+.category__books-body {
+	display: grid;
+	grid-template-columns: repeat(auto-fill, minmax(var(--width-card-large), 1fr));
+	gap: var(--gap-regular);
+}
+
+.category__book-image {
+	width: 100%;
+	aspect-ratio: 2 / 3;
+	margin-bottom: var(--gap-small);
+	border-radius: var(--border-radius-medium);
+	border: 1px solid var(--shadow-color-small);
+}
+
+/* Filter styles */
+
+.filter,
+.filter__inputs {
+	display: flex;
+	flex-direction: column;
+	gap: var(--gap-medium);
+	padding: 0 var(--outline-tiny);
+}
+
+.filter__title {
+	height: var(--height-sub-title);
+}
+
+.filter__input-group {
+	position: relative;
+}
+
+.search-control,
+.sort-control {
+	display: block;
+	width: 100%;
+	height: var(--height-main-input);
+	border-radius: var(--border-radius-regular);
+}
+
+.filter__input-search-icon,
+.filter__input-sort-icon {
+	position: absolute;
+	top: 50%;
+	transform: translateY(-50%);
+	pointer-events: none;
+}
+
+.filter__input-search-icon {
+	width: var(--icon-size-large);
+	height: var(--icon-size-large);
+	left: var(--gap-regular);
+}
+
+.search-control {
+	background: var(--shadow-color-tiny);
```

```suggestion
	background-color: var(--shadow-color-tiny);
```
Use `background-color`, `background-image` etc. This improves the readability of the code